.github/workflows/osv-scanner-pr.yml

```diff
@@ -0,0 +1,19 @@
+name: OSV-Scanner PR Scan
+
+# Change "main" to your default branch if you use a different name, i.e. "master"
+on:
+  pull_request:
+    branches: [ main ]
+  merge_group:
+    branches: [ main ]
+
+# Declare default permissions as read only.
+permissions:
+  # Only need to read contents
+  contents: read
+  # Require writing security events to upload SARIF file to security tab
+  security-events: write
+
+jobs:
+  scan-pr:
+    uses: "google/osv-scanner/.github/workflows/osv-scanner-reusable-pr.yml@main"
```

Comment:
Can this use an explicit commit so that the version is pinned?

Reply:
It won't be that useful in the preview stage, as the reusable action itself doesn't use a pinned action version (which is really hard to do when the action is in the same repo as the binary). When it's fully released in December in it's own repo, then we'll have specific tagged releases that can be more easily pinned.